webserver/webserver/src/main/java/io/helidon/webserver/HashResponseHeaders.java

```diff
@@ -203,6 +203,24 @@ public void addCookie(SetCookie cookie) {
         add(Http.Header.SET_COOKIE, cookie.toString());
     }
 
+    @Override
+    public void clearCookie(String name) {
+        SetCookie expiredCookie = SetCookie.builder(name, "deleted")
+                .path("/")
+                .expires(ZonedDateTime.of(1970, 1, 1, 0, 0, 0, 0, ZoneId.of("GMT+0")))
```

Comment:
The date instance could be created as a static lazy value

Reply:
Sure, don't think this would be a popular method, but we can cache it